src/main/java/duke/CheeChat.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import duke.command.Command;
+import duke.util.Parser;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+
+public class CheeChat {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    private String filePath = "data/duke.txt";
+
+    public CheeChat() {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        tasks = new TaskList(storage.load());
+    }
+
+    protected String getResponse(String input) {
```

Comment:
I like that these steps are being packaged as one function that complys with the meaning of name of the function.